src/main/java/ch/njol/skript/effects/EffHealth.java

```diff
@@ -34,42 +34,46 @@
 import ch.njol.util.Math2;
 import org.bukkit.entity.Damageable;
 import org.bukkit.event.Event;
+import org.bukkit.event.entity.EntityDamageEvent.DamageCause;
 import org.bukkit.inventory.ItemStack;
 import org.eclipse.jdt.annotation.Nullable;
+import org.jetbrains.annotations.UnknownNullability;
 
 @Name("Damage/Heal/Repair")
 @Description("Damage/Heal/Repair an entity, or item.")
```

Comment:
```suggestion
@Description("Damage, heal, or repair an entity or item.")
```
As well as this, the description should be updated a little bit to explain the damage cause stuff.